modes/longitudinal/src/index.ts

```diff
@@ -90,6 +90,18 @@ function modeFactory({ modeConfiguration }) {
       //   },
       // ]);
 
+      customizationService.addModeCustomizations([
```

Comment:
Example customizations should be included in the basic test mode, not in longitudinal.

Reply:
I was just trying to see if it appends, but it did not